design-guidelines/page-layout-patterns.md

```diff
@@ -0,0 +1,145 @@
+
+### Page header
+
+The page header informs the user which page they're on. It often displays the name of the page, and it may contain content and interactive components that are relevant to the rest of the page.
+
+For more detailed information about page header usage, see the Storybook documentation for the `PageHeader` component.
+
+### Standard layout
+
+The most common page layout has a sidebar, header bar, and a main content area with a page header. The content in the page header and the primary content area stretches to fill the available width.
+
+![standard](./img/page-layout-patterns/standard.png)
+
+#### Primary content area
+
+The primary content area can show anything, but it is most commonly a collection of related objects, or details about one specific object.
+
+##### Primary content actions
+
+If there are actions that directly affect what is displayed in the primary content area, they go before the content they affect, but after the page header.
+
+![standard-withActions](./img/page-layout-patterns/standard-withActions.png)
+
+There are many possibilities for what interactive components can be used in the primary content actions area.
+
+Action buttons are placed flushed to the right edge of the primary content area, and are ordered right-to-left from most important to least important.
+
+![contentAction-oneAction](./img/page-layout-patterns/contentAction-oneAction.png)
+
+![contentAction-twoActions](./img/page-layout-patterns/contentAction-twoActions.png)
+
+If there are more than three actions, the two most important actions are placed to the left of an overflow menu.
+
+![contentAction-overflow](./img/page-layout-patterns/contentAction-overflow.png)
+
+If there are filters, they are flushed to the left edge of the primary content area.
+
+![contentAction-filter](./img/page-layout-patterns/contentAction-filter.png)
+
+If there is a text filter and an additional filter (e.g.: dropdown or segmented control), the text filter comes first.
+
+![contentAction-txtFilter](./img/page-layout-patterns/contentAction-txtFilter.png)
+
+![contentAction-txtFilter+filter](./img/page-layout-patterns/contentAction-txtFilter+filter.png)
+
+![contentAction-txtFilter+filter+actions](./img/page-layout-patterns/contentAction-txtFilter+filter+actions.png)
+
+#### Examples
+
+##### Standard layout
+
+![standard-example-basic](./img/page-layout-patterns/standard-example-basic.png)
+
+![standard-example-withActions](./img/page-layout-patterns/standard-example-withActions.png)
+
+![standard-example-subheaderSummary](./img/page-layout-patterns/standard-example-subheaderSummary.png)
+
+![standard-example-subheaderPromobanner](./img/page-layout-patterns/standard-example-subheaderPromobanner.png)
+
+![standard-example-tabs](./img/page-layout-patterns/standard-example-tabs.png)
+
+##### Primary content actions
```

Comment:
Good to call out why and when put actions here for folks. we take it for granted. 
Something like "Place actions at tab-level when they only to that specific tab."